src/main/resources/db/changelog/init/01-create-competition.yaml

```diff
@@ -0,0 +1,25 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-table-competition
+      author: Aleksandr
+      changes:
+        - createTable:
+            tableName: competition
+            columns:
```

Comment:
Competition can have:
1. start date
2. end date
3. participants
4. status
5. tasks
And potentially more. 

Could you please explore a comprehensive list of fields that could potentially be configured?

Reply:
converted to yml

Reply:
working on csv file format